ra/ra_test.go

```diff
@@ -1860,6 +1861,21 @@ func TestNewOrder(t *testing.T) {
 	test.AssertByteEquals(t, order.Csr, csr)
 	test.AssertEquals(t, *order.Id, int64(1))
 	test.AssertEquals(t, len(order.Authorizations), 3)
+
+	// Reuse all of the existing authorizations from the previous order
```

Comment:
Can you add a test case that reuses some but not all of the previous authorizations? And one with a CSR containing duplicated names, in different case?

Reply:
Ping on this comment